.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: release
```

Comment:
Can you add the `opentelemetry.yml` one? 🙏 

Similar to https://github.com/elastic/apm-pipeline-library/blob/main/.github/workflows/opentelemetry.yml

Reply:
https://github.com/elastic/apm-aws-lambda/pull/362/commits/7b1c80c4bf936c08d00e59c0e8c8ad5096e93ded